sbtix-tool.nix

```diff
@@ -44,6 +44,20 @@ let
       mkdir -p "$SBTIX_GLBASE_DIR/plugins"
     fi
 
+    SBTIX_SETTINGS_FILE="$SBTIX_GLBASE_DIR/sbtix_settings.sbt"
+    if [ -f $SBTIX_SETTINGS_FILE ]; then
+      echo "Resetting $SBTIX_SETTINGS_FILE"
+      rm $SBTIX_SETTINGS_FILE
+    fi
+    if [ -f sbtix-build-inputs.nix ]; then
+      cat >> $SBTIX_SETTINGS_FILE <<EOF
+resolvers ++= Seq(
+  new MavenCache("sbtix-local-dependencies", file("$(nix-build sbtix-build-inputs.nix)")),
+  Resolver.file("sbtix-local-dependencies-ivy", file("$(nix-build sbtix-build-inputs.nix)"))(Resolver.ivyStylePatterns),
```

Comment:
Would be quicker to invoke `nix-build` only once.

Not obvious how this would work with flakes. Maybe it should be configurable how the build inputs are realised.

Reply:
TODO
- record the built path in a variable here and pass it to sbtix, so that it can be removed from the generated files later. (e.g. changed to a placeholder). Otherwise devs on a different architecture may be stuck, or if you always have it in cache, you'll have changes in the `repo.nix` just based on who ran it last.
- make the generated expression build `sbtix-build-inputs.nix` and replace the placeholder.